src/kcas.ml

```diff
@@ -189,8 +189,8 @@ let cas loc before state =
 module Loc = struct
   type 'a t = 'a loc
 
-  let make after =
-    { state = Atomic.make @@ new_state after; id = Id.get_unique () }
+  let make ?(mode_val = Mode.lock_free) after =
```

Comment:
I think `Mode.obstruction_free` should be the default.

Reply:
Also, the label could be just `mode`, i.e. `?(mode = Mode.obstruction_free)`.

Reply:
Will modify them accordingly.